### DISCUSSION TITLE
Transcribed text to subtitle?

### DISCUSSION THREAD

**Discussion by Alexandar-Dev** [openai/whisper#1365] [Q&A]
Whisper AI made the transcribed text. 
I fixed spelling errors, and now I want to create subtitles with timestamps. 
How can I do that? 

Is there a command for it? 
I'm using Whisper from CMD.

**Comment by Purfview** [comment]
But it already creates subtitles with timestamps by default. [for example srt file]

**Comment by EtienneAb3d** [comment]
If you have a corrected text, you now need this to add timestamps to it:
https://github.com/EtienneAb3d/WhisperTimeSync
